recipes/catch2/2.x.x/conandata.yml

```diff
@@ -47,6 +47,9 @@ sources:
   "2.13.7":
     url: "https://github.com/catchorg/Catch2/archive/v2.13.7.tar.gz"
     sha256: "3cdb4138a072e4c0290034fe22d9f0a80d3bcfb8d7a8a5c49ad75d3a5da24fae"
+  "2.13.8":
```

Comment:
All other versions except previous do not use quotes, and I remove quotes around 2.13.7 (as a side activity) in my PR: #8613
So I suggest you remove the quotes also.

Reply:
The update bot is only so smart 🤷